src/KM_Points.pas

```diff
@@ -62,6 +62,8 @@   TKMTriMesh = record
   function KMInRect(aPoint: TKMPointF; aRect: TKMRectF): Boolean; overload;
   function KMRectArea(aRect: TKMRect): Integer;
   function KMRectMove(aRect: TKMRect; X,Y: Integer): TKMRect;
+  procedure KMRectAddPnt(var aRect: TKMRect; X,Y: Integer); overload;
```

Comment:
Should be called "KMRectIncludePoint", since we are including this point into the Rect.